src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,98 @@
+package lotto.model;
+
+import java.util.*;
+
+public class Lotto {
+    private List<Integer> numbers = new ArrayList<>();
+    private static final String WRONG_NUMBER_FORMAT = "잘못된 형식입니다. 숫자 구분은 오직 ,(comma)로만 해주세요.";
+    private static final String NOT_SIX_NUMBERS = "6개의 숫자를 입력(생성)해주세요.";
+    private static final String DOUBLE_ELEMENT_EXCEPTION = "중복된 원소가 존재합니다.";
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        validateUnique(numbers);
+        this.numbers = numbers;
+    }
+
+    public Lotto(String numberCommand) {
+        List<String> commaSeparatedNumbers = List.of(numberCommand.split(","));
+        validateNumber(commaSeparatedNumbers);
+        List<Integer> shiftedNumbers = new ArrayList<>();
+        for (String number: commaSeparatedNumbers)
+            shiftedNumbers.add(Integer.parseInt(number));
+        validate(shiftedNumbers);
+        validateUnique(shiftedNumbers);
+        Collections.sort(shiftedNumbers);
```

Comment:
검증 부분을 제외한 곳은 스트림을 사용하면 더욱 가독성이 좋게 변경할 수 있습니다.
```suggestion
        List<Integer> shiftedNumbers = commaSeparatedNumbers.stream()
                .map(Integer::parseInt)
                .sorted()
                .toList();
```